src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -207,7 +211,9 @@ public Stream ReadBlob(string dataTableName, string dataColumnName, long rowId)
         private Stream ReadBlob_InTransaction(string tableName, string columnName, long rowId)
         {
             const int ReadOnlyFlags = 0;
+#pragma warning disable CA2000 // Dispose objects before losing scope - disposed with sqlite3_blob_close invocation.
```

Comment:
💭 This might not be a case we can handle. It seems like SQLitePCL.raw should support error checking in explicit `Dispose` calls.